.gitignore

```
@@ -8,6 +8,7 @@
 
 # intellij
 .idea/ant.xml
+/.idea/codestream.xml
```

For consistency I guess this one should also be declared relative?
```suggestion
.idea/codestream.xml
```

---

works for me. updated.